build/Dockerfile.ci

```diff
@@ -7,21 +7,14 @@
 
 # We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
 # to install it.
-FROM registry.access.redhat.com/ubi8/ubi-minimal as build
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 as build
 LABEL stage=build
-RUN microdnf -y install rsync make git tar findutils diffutils && microdnf clean all
-RUN mkdir /build/
 
-# Install go 1.14
-RUN mkdir /build/go
-WORKDIR /build/go
-RUN curl https://golang.org/dl/go1.14.7.linux-amd64.tar.gz -L -o go.tar.gz
-# Check sha256
-RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.tar.gz" |sha256sum -c
-RUN tar -C /usr/local -xzf go.tar.gz
 ENV GOBIN=/usr/local/go/bin
 ENV PATH=${PATH}:${GOBIN}
 
+RUN mkdir /build/
```

Comment:
same as above

Reply:
see https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile.ci#L4-L5

Reply:
I don't think is any issue anymore ever since they upgrade the CI farm.  There are places in `Dockerfile.ci` where we are not doing `mkdir`.